libcontainer/container_linux.go

```diff
@@ -1249,7 +1255,7 @@ func (c *linuxContainer) orderNamespacePaths(namespaces map[configs.NamespaceTyp
 			if strings.ContainsRune(p, ',') {
 				return nil, newSystemError(fmt.Errorf("invalid path %s", p))
 			}
-			paths = append(paths, p)
+			paths = append(paths, fmt.Sprintf("%s:%s", configs.NsName(nsType), p))
```

Comment:
i'm still not sure if setting namespace type here is necessary. Seems like something we do with no gain since the namespace type is already "known" (implicitly) ( and result in more code )